example.c

```diff
@@ -0,0 +1,87 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+// declare function here, linker will find this when linked to
+// libx86simdsortcpp.so
+void keyvalue_qsort_float_sizet(float *, size_t *, size_t);
```

Comment:
Should we add a header file for these declarations? They probably should be documented somewhere, definitely if there's no header file for them